web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,40 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+  const localLicensesRaw = loadAllLicenses();
+  const localLicensesFiltered = localLicensesRaw.filter((it) => it.webflowId === undefined);
+  const localLicensesIds = new Set(localLicensesFiltered.map((i) => i.webflowId));
+
+  const webflowLicense = await getAllLicensesFromWebflow();
+
+  return webflowLicense.filter((item) => {
+    const haslicense = localLicensesIds.has(item.fieldData.id);
```

Comment:
[dust] I'd love a better name here for `hasLicense` that more accurately describes what value this boolean result provides.